cachelib/allocator/MMLru.h

```diff
@@ -195,6 +200,9 @@ class MMLru {
     //     lruInsertionPointSpec = 2, we insert at a point 1/4th from tail
     uint8_t lruInsertionPointSpec{0};
 
+    // Whether to use combined locking for withEvictionIterator.
```

Comment:
Move this after `mmReconfigureIntervalSecs`. Meta internal build failed on the initialization order.

Reply:
Done.